cunumeric/array.py

```diff
@@ -3385,9 +3514,9 @@ def tostring(self, order="C"):
         Multiple GPUs, Multiple CPUs
 
         """
-        return self.__array__().tostring(order=order)
+        return self.__array__().tostring(order=order)  # type: ignore
```

Comment:
`tostring` is deprecated at least, but `mypy` says it does not exist. If this is an alias for `tobytes`, can we just call `tobytes` instead?

Reply:
Agreed, I pushed the suggested change.